static-assets/components/cstudio-common/common-api.js

```diff
@@ -2533,21 +2533,27 @@ var nodeOpen = false,
         CSA.Operations.getWorkflowAffectedFiles(params, {
           success: function (content) {
             if (content && content.length) {
-              CSA.Operations._showDialogueView({
-                controller: 'viewcontroller-cancel-workflow',
-                fn: function (oAjaxCfg) {
-                  // because _showDialogueView was designed to load the body from a
-                  // webscript, must simulate the ajax process here
-                  oAjaxCfg.success({ responseText: '' });
-                },
-                callback: function () {
-                  var view = this;
-                  view.setContent(content);
-                  view.on('continue', function () {
-                    doEdit();
-                  });
+              const eventIdSuccess = 'workflowCancellationDialogContinue';
+              CrafterCMSNext.system.store.dispatch({
+                type: 'SHOW_WORKFLOW_CANCELLATION_DIALOG',
+                payload: {
+                  open: true,
+                  workflowAffectedFiles: content,
```

Comment:
1. workflowAffectedFiles => items
2. parseLegacyItemToSandBoxItem

Reply:
1. done
2. The problem is that `parseLegacyItemToSandBoxItem` is expecting certain properties from the legacyItem, and the items from the workflow API only have `browserUri`, `name` and `path`. Should the items be created differently or should the util be updated?